about_regular_expressions.exs

```diff
@@ -0,0 +1,41 @@
+defmodule AboutRegularExpressions do
+  use Koans
+
+  think "A simple regular expression" do
+    regex = ~r/This [W|w]orks!/
+
+    completion = __?
+
+    assert ("This W" <> completion) =~ regex
```

Comment:
I'm not sure I like the "completion" being the missing piece. I'd say maybe just leave out the whole string. What about

```elixir
think "A simple regular expression" do
  regex = ~r/This [W|w]orks!/

  assert __? =~ regex, "it matches the uppercase W"
  assert __? =~ regex, "it matches the lowercase w"
end
```